cmd/istio-cni/main.go

```diff
@@ -224,7 +224,7 @@ func cmdAdd(args *skel.CmdArgs) error {
 				}
 				if !excludePod {
 					log.Infof("setting up redirect")
-					if redirect, redirErr := NewRedirect(ports, annotations); redirErr != nil {
```

Comment:
`ports` is now unused.
Just remove the `ports` variable and modify `getKubePodInfo` to not return the ports. It should make `getK8sPodInfo` quite shorter.

Reply:
Great catch!